tests/integration/_bootstrap.php

```diff
@@ -216,4 +216,21 @@ function __unset($name) {
   }
 }
 
+if (!function_exists('WC')) {
+  class WC_Mailer { // phpcs:ignore
```

Comment:
This code seems a bit surprising to me. We've been working with woocommerce for a long time and we didn't need this. Why do we need it now?